azurerm/data_source_maps_account.go

```diff
@@ -0,0 +1,94 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmMapsAccount() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceMapsAccountRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validate.ApiManagementApiName,
```

Comment:
Could we use a validation function specifically for this property?

Reply:
Looks like I didn't catch this here. I use a lot of existing code to help drive new additions. Will update.